docs/en/guides/nvidia-jetson.md

```diff
@@ -116,6 +119,23 @@ python3 setup.py install --user
 
 Visit [this page](https://forums.developer.nvidia.com/t/pytorch-for-jetson/72048) to access all different versions of PyTorch for different JetPack versions. For a more detailed list on the PyTorch, Torchvision compatibility, please check [here](https://github.com/pytorch/vision).
```

Comment:
SEO best practices calls for descriptive link text, so i.e. instead of Please check [here]() you should say "See the [PyTorch and Torchvision compatibility page]() for more details".